### PR TITLE
#668 enhancement

### PR DESCRIPTION
**Pull request by tommywulove**

### CODE REVIEW

File: src/commands/view/SelectComponent.js [hunk 1]
```diff
@@ -100,7 +100,21 @@ module.exports = {
       this.onFrameScroll(e);
       this.updateAttached();
     }
+    var model = $(trg).data('model');
+    if (typeof model != 'undefined') {
 
+      if (!model.get("selectable")) {
+        var comp = model && model.parent();
```
Comment (artf) [inline src/commands/view/SelectComponent.js]: In this case, I think it makes more sense checking against `hoverable` property

File: src/commands/view/SelectComponent.js [hunk 2]
```diff
@@ -100,7 +100,21 @@ module.exports = {
       this.onFrameScroll(e);
       this.updateAttached();
     }
+    var model = $(trg).data('model');
+    if (typeof model != 'undefined') {
```
Comment (artf) [inline src/commands/view/SelectComponent.js]: Just use `if (model) {`

File: src/commands/view/SelectComponent.js (hunk 1 above)
Comment (tommywulove) [reply src/commands/view/SelectComponent.js]: updated.

File: src/commands/view/SelectComponent.js (hunk 2 above)
Comment (tommywulove) [reply src/commands/view/SelectComponent.js]: updated.

**Pull request merged**